src/qt/bitcoingui.cpp

```diff
@@ -391,10 +391,10 @@ void BitcoinGUI::createActions()
     usedReceivingAddressesAction = new QAction(QIcon(":/icons/" + theme + "/address-book"), tr("&Receiving addresses..."), this);
     usedReceivingAddressesAction->setStatusTip(tr("Show the list of used receiving addresses and labels"));
 
-    openAction = new QAction(QIcon(":/icons/" + theme + "/open"), tr("Open &URI..."), this);
+    openAction = new QAction(QApplication::style()->standardIcon(QStyle::SP_DirOpenIcon), tr("Open &URI..."), this);
```

Comment:
I think it would be nice if we would also:
1) remove references like https://github.com/dashpay/dash/blob/v0.12.1.x/src/qt/dash.qrc#L50, https://github.com/dashpay/dash/blob/v0.12.1.x/src/Makefile.qt.include#L213 etc for all themes
2) remove resources themselves (i.e. file info.png in this case, for all themes too)